my_tuple = ("first","second",3,4,5)
print(my_tuple)
my_tuple[1]= 2

''' error msg
ERROR!
('first', 'second', 3, 4, 5)
Traceback (most recent call last):
  File "<main.py>", line 3, in <module>
TypeError: 'tuple' object does not support item assignment

=== Code Exited With Errors ==='
'''